### PR TITLE
Bug 1787028: fix runtime error creating PVC with no storage class

### PR DESCRIPTION
**Pull request by spadgett**
https://bugzilla.redhat.com/show_bug.cgi?id=1787028

/assign @zherman0

### CODE REVIEW

File: frontend/public/components/storage/create-pvc.tsx [hunk 1]
```diff
@@ -155,29 +155,22 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
   };
 
   const handleStorageClass = (updatedStorageClass) => {
+    const provisioner: string = updatedStorageClass?.provisioner || '';
```
Comment (zherman0) [inline frontend/public/components/storage/create-pvc.tsx]: Does optional chaining work in Firefox or IE?

Comment (spadgett) [reply frontend/public/components/storage/create-pvc.tsx]: It's suppored in TypeScript. The final JS won't have it.

**Comment by benjaminapetersen** [comment]
/lgtm
/cherry-pick release-4.3

Since this was reported on 4.2.z

**Pull request merged**